tests/unit/utils/test_version.py

```diff
@@ -895,13 +897,16 @@ class VersionParams:
     VersionParams('unknown-dist', known_distribution=False),
     VersionParams('no-ssl', ssl_support=False),
     VersionParams('no-autoconfig-loaded', autoconfig_loaded=False),
+    VersionParams('no-config-py-loaded', config_py_loaded=False),
 ], ids=lambda param: param.name)
-def test_version_output(params, stubs, monkeypatch):
+def test_version_output(params, stubs, monkeypatch, config_stub):
     """Test version.version()."""
     class FakeWebEngineSettings:
         default_user_agent = ('Toaster/4.0.4 Chrome/CHROMIUMVERSION '
                               'Teapot/4.1.8')
 
+    config.instance = config_stub
```

Comment:
This wasn't needed, as the `config_stub` fixture automatically patches `config.instance` already:

https://github.com/qutebrowser/qutebrowser/blob/51c18111c54c7b5fb57d1522a2faf764386071ad/tests/helpers/fixtures.py#L307

Also, in general such patching should be done using pytest's monkeypatch fixture:

https://pytest.readthedocs.io/en/latest/monkeypatch.html

That way, it's automatically undone when the test is finished, so that no other tests are influenced by the change.

I removed it while merging!

Reply:
I will remember it for the future.